File: java-starter/src/main/java/org/example/Application.java
package org.example;

import io.nitric.Nitric;

public class Application {
    public static void main(String[] args) {
        var api = Nitric.INSTANCE.api("api");

        api.get("/hello/:name", (ctx) -> {
            var name = ctx.getReq().getParams().get("name");

            ctx.getResp().text("hello " + name);

            return ctx;
        });

        Nitric.INSTANCE.run();
    }
}

File: kotlin-starter/src/main/kotlin/Application.kt
import io.nitric.Nitric

fun main() {
    val api = Nitric.api("api")

    api.get("/hello/:name") { ctx ->
        val name = ctx.req.params["name"]

        ctx.resp.text("hello $name")

        ctx
    }
    Nitric.run()
}